omod/src/main/resources/config.xml

```diff
@@ -27,6 +27,7 @@
     <require_modules>
         <require_module version="${uiframeworkVersion}">org.openmrs.module.uiframework</require_module>
         <require_module version="${uilibraryVersion}">org.openmrs.module.uilibrary</require_module>
+        <require_module version="${webservicesRestVersion}">org.openmrs.module.webservices.rest</require_module>
```

Comment:
I guess we don't technically have to require this, but at this point it's pretty much always there

Reply:
Not sure how this worked without this actually.  We are depending on a class from this module, it's only by luck or some transitive dependency that this worked earlier without this.  We definitely need it.